src/systems/filecoin_vm/interpreter/vm_registry.go

```diff
@@ -63,22 +65,22 @@ func init() {
 func _registerBuiltinActors() {
 	// TODO
 
-	cron := &sysactors.CronActorCode_I{}
+	cron := &cronact.CronActorCode_I{}
 
-	RegisterActor(InitActorCodeID, &sysactors.InitActorCode_I{})
+	RegisterActor(InitActorCodeID, &initact.InitActorCode_I{})
 	RegisterActor(CronActorCodeID, cron)
-	RegisterActor(AccountActorCodeID, &sysactors.AccountActorCode_I{})
-	RegisterActor(StoragePowerActorCodeID, &spc.StoragePowerActorCode_I{})
-	RegisterActor(StorageMarketActorCodeID, &market.StorageMarketActorCode_I{})
```

Comment:
Do we need to register BurntFund and Reward actors here too?

Reply:
Yes we probably do, but I'm trying to avoid changes, including fixes, while doing this move. This piece in particular might change anyway as we hook up execution scaffolds.